hypertrace-view-generator/hypertrace-view-generator/src/test/resources/configs/all-views/application.conf

```diff
@@ -3,7 +3,7 @@ service.admin.port = 8099
 
 main.class = org.hypertrace.core.viewgenerator.service.MultiViewGeneratorLauncher
 
-view.generators = ["view-gen-backend-entity", "view-gen-raw-service", "view-gen-raw-traces", "view-gen-service-call", "view-gen-span-event"]
+view.generators = ["view-gen-backend-entity", "view-gen-raw-service", "view-gen-raw-traces", "view-gen-service-call", "view-gen-span-event", "view-gen-log-event"]
```

Comment:
will this need any change in `helm` files?

Reply:
Done